### ISSUE TITLE
Remove (optional) from innoextract dependency

### ISSUE THREAD

**Issue by stzokev** [coderobe/VBiosFinder#11]
If I don't have innoextract  I get the 
 /var/lib/gems/2.3.0/gems/cocaine-0.5.8/lib/cocaine/command_line.rb:82:in `run': Cocaine::CommandNotFoundError (Cocaine::CommandNotFoundError)

which others have reported before. So apparently it is not optional as indicated in the instructions. Or if it is optional in some cases, a clarification might be needed.

Great tool otherwise! Thanks for your hard work!

**Comment by coderobe** [comment]
Specifically, though, it will tell you
```
checking for innoextract... no
Install 'innoextract' on your system (required for Inno Installers)
```

**Comment by stzokev** [comment]
Yes, but it gives similar error messages for all dependencies, and half of them are marked as optional. Knowing that, I just ignore these messages.
